lerobot/scripts/visualize_dataset.py

```diff
@@ -236,8 +236,8 @@ def create_video(id, src):
     first_frame_idx = dataset.episode_data_index["from"][ep_index].item()
 
     for key in dataset.video_frame_keys:
-        # Example of video_path: 'videos/observation.image_episode_000004.mp4'
-        video_path = dataset.hf_dataset.select_columns(key)[first_frame_idx][key]["path"]
+        # Example of video_path: 'https://huggingface.co/datasets/{dataset.repo_id}/resolve/{dataset.version}/videos/observation.image_episode_000004.mp4'
+        video_path = f"https://huggingface.co/datasets/{dataset.repo_id}/resolve/{dataset.version}/" + dataset.hf_dataset.select_columns(key)[first_frame_idx][key]["path"]
```

Comment:
Super cool that we have this way to visualize mp4 hosted on the hub, but is there a way to visualize mp4 stored locally? It's a use case we want to support.

What's weird is that when I wait enough so that the video can load, the slider thing works locally :/


https://github.com/huggingface/lerobot/assets/4681518/5c39e8df-bec1-4ecd-a6b4-72a3ae92eaef